src/indexer/index.ts

```diff
@@ -1,3 +1,9 @@
+import algodClient from "@/config/algob.config";
+import {
+	Key,
+	StateValue,
+	AccountAddress,
+} from "@algo-builder/algob/build/types";
```

Comment:
I don't think import from `build/types` is good approach.

Reply:
okay, I will create an interface here itself